.github/workflows/build.yaml

```diff
@@ -64,3 +64,23 @@ jobs:
           name: flake-checker-X64-Linux
           path: result/bin/flake-checker
           retention-days: 1
+
+  build-artifacts-ARM64-Linux:
+    runs-on: ubuntu-22.04
```

Comment:
Does this really work...? I would expect this to continue using the x86_64 ubuntu image, not an arm64 one...

Reply:
uhm.